bean-validation/4.0/_index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Jakarta Validation 4.0 (Under development)"
+date: 2024-03-27
+summary: "Release for Jakarta EE 11"
+---
+Jakarta Validation defines a metadata model and API for JavaBean and method validation.
+
+This release is targeting Jakarta EE 12 and will look to introduce the new features, possibly deprecate some of the existing features, .
+
+### New features, enhancements or additions
+<!-- List here -->
+* - Actually deprecate the methods in `ConstraintViolationBuilder`,
```

Comment:
Having the dash here has this get represented as a double bullet point
```suggestion
* Actually deprecate the methods in `ConstraintViolationBuilder`,
```

Reply:
Hey, thanks for taking a look! I've addressed your suggestions in the PR mentioned above.